exercises/concept/paulas-palindromes/src/Main.elm

```diff
@@ -0,0 +1,52 @@
+module Main exposing (..)
+
+import Browser
+import Html exposing (Html, div, input, text)
+import Html.Attributes exposing (placeholder, value)
+import Html.Events exposing (onInput)
```

Comment:
Not sure if we need to import all of this in the stub.

Reply:
Makes it a bit easier for the student though, right? And there isn't much gained by forcing the student to type it in ...